src/microbe_stage/editor/EditorActionHistory.cs

```diff
@@ -191,6 +175,37 @@ public bool HexPlacedThisSession<THex>(THex hex)
         return History.OfType<HexPlacementActionData<THex>>().Any(a => a.PlacedHex == hex);
     }
 
+    /// <summary>
+    ///   Pick the data with which you can regain maximum MP to combine.
+    /// </summary>
+    /// <param name="currentData">The data you want to combine</param>
+    /// <param name="previousData">A list of data <see cref="currentData"/> may combine with</param>
+    /// <returns>
+    ///   Three-element tuple:
+    ///   CostDelta is non-positive, indicating the amount of MP you will regain;
+    ///   MinimumCostActionData is the action data to combine with (not yet combined);
+    ///   Mode is the interference mode currentData has with MinimumCostActionData.
+    /// </returns>
+    private static (int CostDelta, EditorCombinableActionData MinimumCostActionData, ActionInterferenceMode Mode)
+        GetLeastCostCombineTuple(EditorCombinableActionData currentData,
```

Comment:
`FindCheapestActionToCombineWith` 
I think a name like that would make this clearer as to what this does.

Reply:
Fine. Wait for a minute.